modules/pam_timestamp/pam_timestamp_check.8.xml

```diff
@@ -72,7 +72,7 @@ see if the default timestamp is valid, or optionally remove it.
             <para>
                By default <command>pam_timestamp_check</command> checks or removes
                timestamps generated by <emphasis>pam_timestamp</emphasis> when
-               the user authenticates as herself. When the user authenticates as a
+               users authenticate as themselves. When users authenticate as a
```

Comment:
I suggest the following wording:
`users authenticate as themselves. When the user authenticates as a`